### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by BenGitsCode**
Update from diagnostic template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,22 @@ and other [FAQs](https://github.com/ga-wdi-boston/meta/wiki/) related to
...
 and [pull requests](https://github.com/ga-wdi-boston/meta/wiki/PullRequest).
 
+You may use **any resource** other than each other to complete this diagnostic.
+This includes referencing talk materials, appropriate documentation, and
+searching for help online.
+
+You should be running `grunt nag` before diagnosing any bugs, since it finds
+some of the most common sources of errors. After `grunt nag` passes, you should
+run `grunt test` to run the included tests. Tests will tell you whether of not
+your responses are correct.
```
Comment (jrhorn424) [inline README.md]: Comfort/clarity check is missing (around this line? double check me on this). Please address this.

File: README.md [hunk 2]
```diff
@@ -19,15 +19,11 @@ You may use **any resource** other than each other to complete this diagnostic.
 This includes referencing talk materials, appropriate documentation, and
 searching for help online.
```
Comment (jrhorn424) [inline README.md]: Try to include changes like this in the previous commit next time. I consider things like this part of updating from template. Don't worry about addressing it this time.

**Review by jrhorn424 — CHANGES_REQUESTED** [review]

**Comment by jrhorn424** [comment]
Please address also my comments on #108 here, on this PR.

File: diagnostic.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ What is the command to start an instance of a rails server?
 // your response here
 ```
 
-What are the commands to drop, create and migrate a database from the command
+What are the commands to drop, create, migrate and seed a database from the command
 line? (3 bullet points)
```
Comment (jrhorn424) [inline diagnostic.md]: In the solution, this line says (5 bullet points), here it only says (3 bullet points). Please address.

**Review by jrhorn424 — CHANGES_REQUESTED** [review]

File: diagnostic.md (hunk 3 above)
Comment (BenGitsCode) [reply diagnostic.md]: Ah, I'm sorry I missed that. Fixed in 3adac871a8eb5fa4406bc3a4311a798176745999

**Comment by jrhorn424** [comment]
Awesome commit message. ;)

**Pull request merged**